data/query.go

```diff
@@ -78,6 +79,12 @@ func SetDefaultQueries(ctx context.Context, url *url.URL) (*url.URL, *Pagination
 		query.Set("limit", DefaultLimitStr)
 		limit = DefaultLimit
 	} else {
+		// TO MAKE NEAT
+		if limit > cfg.DefaultMaximumLimit {
+			limit = cfg.DefaultMaximumLimit
```

Comment:
Your code coverage / unit tests missed this line.
(they seem to hit nearly everything else, which is great)